README.md

```diff
@@ -31,7 +31,7 @@ To learn more about OpenThread, visit [openthread.io](https://openthread.io).
 
 ## Getting started
 
-See [OT Commissioner CLI](src/app/cli/README.md) to get started.
+See [OT Commissioner guide](https://openthread.io/guides/commissioner) to get started. Reference to [OT Commissioner CLI documentation](src/app/cli/README.md) for all CLI commands.
```

Comment:
Second sentence can be

"See [OT Commissioner CLI documentation](src/app/cli/README.md) for a reference of all CLI commands."

Reply:
Resolved, please review @Vyrastas